lib/src/native/realm_core.dart

```diff
@@ -384,6 +384,9 @@ class _RealmCore {
     });
   }
 
+  void clear(RealmList list) {
```

Comment:
```suggestion
  void listClear(RealmList list) {
```

Reply:
dart does not have overloads so this might clash with something else in the future.

Reply:
I RealmList this method clear is inherited from ListBase and it is overridden. If we have listClear, then how the implementation of clear will be? And also if we have two methods for clear it is better they to start with clear (clear and clearList), so that the developer will see them both side by side in suggestion list. But I think it is good to be just clear.